feat(search): rest api overview

weakish opened the pull request.

weakish committed:
feat(search): rest api overview

leeyeh commented:
Deploy preview for *lc-docs-en* ready!

Built with commit 5faf2d2c74001057b0aa0289dbed4e09d3025971

https://deploy-preview-168--lc-docs-en.netlify.com

weakish committed:
feat(search): full text search

weakish committed:
feat(search): GeoPoints sorting

weakish committed:
feat(search): more like this query

weakish committed:
feat(search): enable search indexes

weakish committed:
feat(search): remove word segmentation section

This section is for Chinese only.
English strings do not need word segmentation.

weakish committed:
feat(search): add link on index page

weakish committed:
fix(search): typos and markup

weakish committed:
Merge branch 'master' into rest-api-search

weakish committed:
fix: nav links

Commit:
fix: wording
